arch/arm/boot/dts/nxp/imx/imx6ul-ts7250v3.dtsi

```diff
@@ -34,7 +34,6 @@
 		gpio9 = &fpga_bank4;
 		gpio10 = &fpga_bank5;
 		gpio11 = &pc104gpio;
-		i2c5 = &i2c5_gpio;
```

Comment:
Changes to this file are basically reverting the previous commit, 18e58a0998d76abf8e26dce446128e543851366c. If the 18e58a0998d76abf8e26dce446128e543851366c commit is not necessary, it should be squashed so that the lines are not added then removed in two back to back commits